frontend/shared/ticket-builder/index.ts

```diff
@@ -91,7 +90,7 @@ export class TicketBuilder {
             }
         });
 
-        const fileName = (this.tickets.length == 1 ? Formatter.fileSlug(this.tickets[0].getTitle() + (this.tickets[0].getIndexText() ? (' ' + this.tickets[0].getIndexText()) : '')) : Formatter.fileSlug('Tickets ' + this.webshop.meta.name)) + '.pdf';
+        const fileName = ((this.tickets.length === 1 ? Formatter.fileSlug(this.tickets[0].getTitle() + (this.tickets[0].getIndexText() ? ('_' + this.tickets[0].getIndexText()) : '')) : Formatter.fileSlug('Tickets_' + this.webshop.meta.name)) + '.pdf').replace(/\s/g, '_');
```

Comment:
Het is netter om hier gewoon een andere formatter te gebruiken (ook - is doorgaans iets netter dan _, maar dat is misschien persoonlijk)

```suggestion
        const fileName = ((this.tickets.length === 1 ? Formatter.fileSlug(this.tickets[0].getTitle() + (this.tickets[0].getIndexText() ? ('_' + this.tickets[0].getIndexText()) : '')) : Formatter.slug('Tickets ' + this.webshop.meta.name)) + '.pdf');
```